controls/radwindow/features/window-icon.md

```diff
@@ -41,9 +43,11 @@ This will be the final result:
 
 ![Rad Window Features Window Icon 01](images/RadWindow_Features_Window_Icon_01.png)
 
-If you want to share a common icon layout structure for the content of multiple windows, define an appropriate __DataTemplate__ and set it to the __IconTemplate__ property of the __RadWindow__.
+## IconTemplate Property
+
+If you want to share a common icon layout structure for the content of multiple windows, define an appropriate `DataTemplate` and set it to the `IconTemplate` property of the RadWindow.

```

Comment:
I would remove the "If you want to" part from the texts in the article. This is an instructional documentation and in my opinion most of the time you can directly say what should be done. For example: "To share a common icon layout......., set the .... property". 

"If you want to share a common icon layout structure for the content of multiple windows"
This information is not necessary. Sharing the layout between multiple controls is a main feature of DataTemplate, but I think the focus here should be that the template property can be used to customize the visual appearance of the icon content.